CONTRIBUTING.rst

```diff
@@ -921,6 +921,18 @@ should not wait until a minor release. The steps for a patch release
    from conda and follow the directions in step 10 in the minor release
    section.
 
+Dependency version policy
+-------------------------
+
+Python and VTK dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We support all supported `Python versions`_ and `VTK versions`_ that
+support those Python versions. As much as we would prefer to follow
+`NEP 29`_, we follow the VTK version policy as an interface library of VTK.
```

Comment:
I can't find a VTK version policy.  Maybe this can be changed to "..., we follow VTK versions as an interface library of VTK."

Gets messy to have a GitHub suggestion with two different concepts in it, so just leaving this one in a comment.

Reply:
Thanks!